cmd/prow-job-dispatcher/main.go

```diff
@@ -529,25 +529,59 @@ func clustersMapToSet(clusterMap ClusterMap) sets.Set[string] {
 	return clusterSet
 }
 
-func createPR(o options) {
+func gitCloneRelease() error {
+	cmd := exec.Command("git", "clone", "--depth", "1", "--single-branch", "git@github.com:openshift/release.git")
```

Comment:
We could probably use `prow's` `gitClient` here to do this, but if this would be the only usage, I think this is fine also

Reply:
only and actually working pretty well, so I would keep it